source/slang/slang-ir-insts.h

```diff
@@ -422,6 +422,15 @@ struct IRRequireGLSLExtensionDecoration : IRDecoration
     UnownedStringSlice getExtensionName() { return getExtensionNameOperand()->getStringSlice(); }
 };
 
+struct IRRequireWGSLExtensionDecoration : IRDecoration
+{
```

Comment:
I feel like we may want something like `RequireTargetExtension(glsl, "...")` instead of having one op code per target. But we don't need to change this in this PR.